examples/using-sdk/helper/env/main.go

```diff
@@ -0,0 +1,37 @@
+/*
+ * .-'_.---._'-.
+ * ||####|(__)||   Protect your secrets, protect your business.
+ *   \\()|##//       Secure your sensitive data with Aegis.
+ *    \\ |#//                    <aegis.ist>
+ *     .\_/.
+ */
+
+package main
+
+import (
+	"fmt"
+	"github.com/shieldworks/aegis/sdk/sentry"
+	"strings"
+)
+
+func main() {
+	d, err := sentry.Fetch()
+	if err != nil {
+		msg := err.Error()
+
+		if strings.Contains(msg, "Secret does not exist") {
```

Comment:
This is hacky, but this is just sample code; not part of the library, so I can live with this for now.